subiquity/controller.py

```diff
@@ -90,15 +86,15 @@ def make_autoinstall(self):
         return {}
 
 
-class NoUIController(SubiquityController):
-
-    def start_ui(self):
-        raise Skip
-
-    def cancel(self):
-        pass
+class SubiquityTuiController(SubiquityController, TuiController):
 
     def interactive(self):
+        if not self.app.autoinstall_config:
+            return True
+        i_sections = self.app.autoinstall_config.get(
+            'interactive-sections', [])
+        if '*' in i_sections or self.autoinstall_key in i_sections:
```

Comment:
You can just return this conditional rather than using an if blah return True else return False construct